src/index.html

```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <div class="from-field">
+          <label
+            >Surname:
+            <input type="text"
+                   name="surname"
+                   autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >Name:
+            <input type="text"
+                   name="name"
+                   autocomplete="off"
+            >
+          </label>
+        </div>
```

Comment:
Apply all over the page
1) no needs to wrap each field int a `div` you can style `label` directly
2) apply code styling recommendation
```suggestion
          <label class="form-field">
            Name:
            <input 
              type="text"
              name="name"
              autocomplete="off"
            >
          </label>
```